src/plugins/tables/tables.js

```diff
@@ -135,31 +135,42 @@ $document.on( "init.dt draw.dt", selector, function( event, settings ) {
 		ol = document.createElement( "OL" ),
 		li = document.createElement( "LI" );
 
-	// Update Pagination List
-	for ( var i = 0; i < paginate_buttons.length; i++ ) {
-		var item = li.cloneNode( true );
-		item.appendChild( paginate_buttons[ i ] );
-		ol.appendChild( item );
-	}
+	// Determine if Pagination required
+	if ( paginate_buttons.length < 4 ) {
+		$( ".dataTables_paginate" ).addClass( "hidden" );
+		$( ".dataTables_paginate" ).attr( "aria-hidden", "true" );
+	} else {
+
+		// Make sure Pagination is visible
+		$( ".dataTables_paginate" ).removeClass( "hidden" );
+		$( ".dataTables_paginate" ).attr( "aria-hidden", "false" );
+
+		// Update Pagination List
+		for ( var i = 0; i < paginate_buttons.length; i++ ) {
+			var item = li.cloneNode( true );
```

Comment:
This are might be not supported in IE11, we will need to do some testing

Reply:
This is not my code and is the existing code of WET. I just see lines 147-148.